### PR TITLE
ExportOptions.plist & Provisioning Profiles Subfolder functionality A…

### PR DESCRIPTION
**Pull request by BhaumikLion**
…dded

### CODE REVIEW

File: Packages/AutomatedBuilding/Editor/Builder/IOSBuilder.cs [hunk 1]
```diff
@@ -28,8 +28,12 @@ private static IOSBuildSettings iosBuildSettings
             get
             {
                 if (_iosBuildSettings == null)
-                    //_iosBuildSettings = AssetDatabase.LoadAssetAtPath<IOSBuildSettings>(IOS_SETTINGS_PATH);
+                {
                     _iosBuildSettings = LionSettingsService.GetSettings<AutoBuilderSettings>().iOS;
+                    // Debug.Log("Settings is not null anymore.." + _iosBuildSettings);
```
Comment (jquentin-lion) [inline Packages/AutomatedBuilding/Editor/Builder/IOSBuilder.cs:33]: No dead code. Uncomment or remove.

File: Packages/AutomatedBuilding/Editor/Builder/IOSBuilder.cs [hunk 2]
```diff
@@ -288,38 +293,53 @@ public static void CreateExportPlist()
...
+            bool directoryExists = Directory.Exists(PROVISIONING_PROFILE_PATH);
+
+            if (!directoryExists)
+            {
+                Directory.CreateDirectory(PROVISIONING_PROFILE_PATH);
+            }
+
             bool fileExists = File.Exists(PROVISIONING_PROFILE_PATH + ProvisionalProfileName + ".mobileprovision");
-            Debug.Log("Path for Provisioning Profile: " + PROVISIONING_PROFILE_PATH + ProvisionalProfileName + ".mobileprovision");
+            //Debug.Log("Path for Provisioning Profile: " + PROVISIONING_PROFILE_PATH + ProvisionalProfileName + ".mobileprovision");
```
Comment (jquentin-lion) [inline Packages/AutomatedBuilding/Editor/Builder/IOSBuilder.cs:342]: No dead code. Uncomment or remove.

**Review by jquentin-lion — CHANGES_REQUESTED** [review]

File: Packages/AutomatedBuilding/Editor/Builder/IOSBuilder.cs (hunk 1 above)
Comment (BhaumikLion) [reply Packages/AutomatedBuilding/Editor/Builder/IOSBuilder.cs]: removed

File: Packages/AutomatedBuilding/Editor/Builder/IOSBuilder.cs (hunk 2 above)
Comment (BhaumikLion) [reply Packages/AutomatedBuilding/Editor/Builder/IOSBuilder.cs]: removed

**Review by jquentin-lion — APPROVED** [review]

**Pull request merged**